wp/plugin/README.md

```diff
@@ -225,6 +225,53 @@ case NAV:
 And a similar invocation of bap will indeed give `UNSAT`, meaning that
 the return values are always identical for identical inputs.
 
+This plugin can also supplement current binary diffing tools to not 
+only identify code reuse between binaries but also understand the 
+implications and causes of their nuances.
```

Comment:
I'm not sure "nuances" is the right word here, maybe "nuanced differences in behavior"?

Reply:
That's true. Just "nuances" is too vague. I won't want to leave room for misinterpretation or misunderstanding. I will change it to "nuanced differences in behavior".